File: Leetcode/two-sum.cpp
// https://leetcode.com/problems/two-sum/submissions/
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        vector<int>s;
        for(int i =0; i < nums.size()-1; i++)
        {
            for(int j = i+1; j < nums.size(); j++)
            {
                if(nums[i] + nums[j] == target)
                {
                    s.push_back(i);
                    s.push_back(j);
                    return s;
                }
            }
        }
        return s; 
    }
};

File: Leetcode/two_sum.py
"""
Given an array of integers nums and an integer target, 
return indices of the two numbers such that they 
add up to target.

You may assume that each input would have exactly 
one solution, and you may not use the same element 
twice.
"""
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        for i in range(len(nums)):
            for j in range(i + 1, len(nums)):
                if nums[j] == target - nums[i]:
                    return [i, j]